src/QuickCheckVEngine/RVFI_DII/DII.hs

```diff
@@ -49,13 +50,14 @@ module QuickCheckVEngine.RVFI_DII.DII (
   DII_Packet
 , diiInstruction
 , diiEnd
+, diiVersNegotiate
+, diiRequestVers
 ) where
 
 import Data.Word
 import Data.Binary
 import Data.Binary.Put
 import Data.Binary.Get
-import Data.Semigroup -- Should no longer be required with modern ghc
```

Comment:
Last time I asked, I think maybe Jon was still keen to have this left in there... I am definitly pro getting rid of it, but maybe we want to check...

Reply:
Just removed it because it triggered a compiler warning. Will revert the change if it's still needed.